release/v1alpha1/types.go

```diff
@@ -0,0 +1,171 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// ReleasePayload encapsulates the information for the creation of the ReleasePayload
+// and aggregates the outputs of the verification tests.
+// The release-controller monitors a specific set of imagestreams for updates.  If/when
+// an update occurs to these imagestreams, the release-controller will:
+//   1) Create a new ReleasePayload
+//      a: in the namespace configured by the --release-namespace parameter of the release-controller
+//      b. named after the name or mirrorPrefix in the aforementioned monitored imagestreams
```

Comment:
does this mean that the same imagestream can have multiple releasepayload objects created for it?  

Please expand this with an example using real names.